README.md

```diff
@@ -58,6 +58,7 @@ There are several scripts available for maintainers:
 - `make fix` -- Perform automatic fixes where possible
 - `make update-auto` -- Run automatic updates from external sources, executed regularly (GitHub action)
 - `make add-token token=c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53` -- Create `info.json` file as asset template.
+- `make add-tokenlist-extended token=c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53` -- Adds a token to tokenlist-extended.json.
```

Comment:
I think we need to support both command here:
`make add tokenlist token=c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53`
`make add tokenlist-extended token=c60_t0x4Fabb145d64652a948d72533023f6E7A623C7C53`

Reply:
added